pulsar-io/file/src/main/java/org/apache/pulsar/io/file/FileSourceConfig.java

```diff
@@ -112,6 +112,12 @@
      */
     private Integer numWorkers = 1;
 
+    /**
+     * If set, do not delete but only rename file that has been processed.
+     * This config only work when 'keepFile' property is false.
+     */
```

Comment:
Please add validation for this rule in the validate method of this class.

Reply:
@lhotari Fixed.

Reply:
@casuallc there are some checkstyle issues:
```
Error:  src/main/java/org/apache/pulsar/io/file/FileSourceConfig.java:[174] (sizes) LineLength: Line is longer than 120 characters (found 129).
Error:  src/main/java/org/apache/pulsar/io/file/FileListingThread.java:[188] (sizes) LineLength: Line is longer than 120 characters (found 129).
Error:  Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.1.2:check (checkstyle) on project pulsar-io-file: You have 2 Checkstyle violations. -> [Help 1]
```